amazon-dynamodb-quickstart/src/main/docker/Dockerfile.jvm

```diff
@@ -7,48 +7,91 @@
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/amazon-dynamodb-jvm .
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/amazon-ssm-quickstart-jvm .
 #
 # Then run the container using:
 #
-# docker run -i --rm -p 8080:8080 quarkus/amazon-dynamodb-jvm
+# docker run -i --rm -p 8080:8080 quarkus/amazon-ssm-quickstart-jvm
```

Comment:
These changes are off.

How did you update these Dockerfiles? We should be updating them automatically on release.